site/content/blog/carvel-in-july-2022.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Carvel In July"
+slug: carvel-in-july-2022
+date: 2022-08-04
+author: Joe Kimmel, Praveen Rewar, and Carvel Engineers
+excerpt: "Like Christmas in July but with Carvel"
+image: /img/logo.svg
+tags: ['kapp', 'impgkg', 'secretgen-controller']
+---
+# ~~Christmas~~ Carvel in July
+
+Ah, July in America[^1]! Long days, warm nights, fireworks, barbecues, vacations, and many iconic releases from our Carvel summer collection. Let us peruse the highlights, as we make s’mores in our bonfires.
+
+
+### Kapp
+
+Kapp celebrated its 50th release with luminous features such as:
+
+
+* default change groups and change rules for kapp-controller resources
+* Output is now available in delicious yaml, just like mom used to make!
+    * Use --diff-changes-yaml while deploying an app to see the complete yaml after rebase rules etc,. are applied.
+* unblockChanges can be used with conditions in waitRules to unblock any waiting changes
+    * These conditions are treated as non success/failure conditions. Helpful when you want to unblock waiting changes but would still want kapp to wait for the resource to reconcile based on other success/failure conditions. 
+
+
+### Imgpkg
+
+* Improved support for cross-platform transfers of bundles
+
+
+### Ytt
+
+Have you ever seen bad data just make a mess out of everything? No longer must you cower in fear that your data might be invalid!
+Go forth boldly, young padawan, ensuring that your data is valid with our new feature[^2], available starting in ytt 0.42.0.
```

Comment:
```suggestion
Go forth boldly, young padawan, ensuring that your data is valid with our new feature, available (in experimental mode) starting in ytt 0.42.0.
```

rationale: 
- John thinks tucking the "experimental" mode of the feature in a footnote is just a step too far away; folks will miss that, he contends.
- save some thunder for when the feature goes "Generally Available"???